### DISCUSSION TITLE
Lesson 7: revert error

### DISCUSSION THREAD

**Discussion by cell0901** [Cyfrin/foundry-full-course-cu#714] [Q&A] [answered]
while running forge test i got this error
```
Failing tests:
Encountered 1 failing test in test/FundMeTest.t.sol:FundMeTest
[FAIL. Reason: EvmError: Revert] testPriceFeedIsAccurate() (gas: 10114)
```
i have added my SEPOLIA RPC URL in .env. And check everything  i can
It works only when i replace mine address with this "0x694AA1769357215DE4FAC081bf1f309aDC325306", which is of patrick's ig.
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/142923689/b3ea71c7-671a-4c08-bc58-386c8e95ba0b)

**Comment by theirrationalone** [comment, marked as answer]
You still dont't have any way to deal with the things that are not available locally i.e., priceFeeds.... You still need to have mocks to deal with it... 

The tests yet, would only work for sepolia rpc Or fork mainnet, testnets.. 

I would advice you to please follow patrick thoroughly!

**Reply by cell0901** [reply]
so i should ignore the errors? And follow ?

**Reply by theirrationalone** [reply]
Yes of course!
